Utils.fs

```diff
@@ -280,6 +280,12 @@ let foldFastTerm (f : 'State -> 'T -> 'State option)
 
     fft f s
 
+/// Repeatedly apply f until a fixed point is reached
```

Comment:
dunno, 1000 or something? High enough that it's probably a loop if we ever see it. NB: there may be a less dumb way of doing this.